cmd/ci-operator/main.go

```diff
@@ -814,3 +827,12 @@ func shorten(value string, l int) string {
 	}
 	return value
 }
+
+func eventRecorder(kubeClient *coreclientset.CoreV1Client, namespace string) record.EventRecorder {
```

Comment:
You should only create one of these and then pass it down to whatever we use.  Don't create it multiple times.